tools/dsctest/src/main.rs

```diff
@@ -65,6 +67,17 @@ fn main() {
             }
             input
         },
+        SubCommand::InDesiredState { input } => {
+            let mut in_desired_state = match serde_json::from_str::<in_desired_state::InDesiredState>(&input) {
+                Ok(in_desired_state) => in_desired_state,
+                Err(err) => {
+                    eprintln!("Error JSON does not match schema: {err}");
+                    std::process::exit(1);
+                }
+            };
+            in_desired_state.value = "SomethingElse".to_string();
```

Comment:
For the sake of the test, when `_inDesiredState` is false, should the `in_desired_state.value` stay the same?

Reply:
In this case, the intent is to just trust the resource if it includes `_inDesiredState` regardless of the values differ.